nss-day-cohort-48/rare-api-orchestra-outlaws
# BACKEND: Tags and Comments

#### Changes Made


1. Added comments and tags models as well as ViewSets
2. Modified views folder to include viewsets for comments and models. They include ability to view all, delete, post, and edit.
3. Currently do not have 


#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout slm-rare-api
   ensure you have loaded all necessary data.
   ```
2. Ensure the backend and client are both running.
4. Test app functionality.

   > Example: In Postman, run GET (for all and by ID), DELETE, PUT, POST for comments and tags. Ensure it works as it should.

5. View code file.
   > Confirm file modifications are present as indicated above.
   > Confirm no unused code or extraneous comments exist.
